plugins/phoenix/__init__.py

```diff
@@ -1,5 +0,0 @@
-"""Phoenix AI Observability Plugin for Neuro SAN Studio."""
-
-from plugins.phoenix.phoenix_init import initialize_phoenix_if_enabled
-
-__all__ = ["initialize_phoenix_if_enabled"]
```

Comment:
The code lives in a Class now, instead of functions. It's better to import the class instead of the package.